lib/treatments.js

```diff
@@ -24,14 +59,15 @@ function storage (env, ctx) {
           pbTreat.notes = obj.notes;
         }
 
-        api( ).insert(pbTreat, function() {
-          //nothing to do here
+        query.created_at = pbTreat.created_at;
+        api( ).update(query, pbTreat, {upsert: true}, function complete (err) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Function 'complete' is already declared in line 49.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1722678008/issues/source?bid=2452379&fileBranchId=3051501#l63)